solid_queue.gemspec

```diff
@@ -44,4 +44,8 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "pg"
   spec.add_development_dependency "sqlite3"
   spec.add_development_dependency "rubocop-rails-omakase"
+  spec.add_development_dependency "ostruct"
+  spec.add_development_dependency "logger"
+  spec.add_development_dependency "fiddle"
```

Comment:
@rosa 

Because without it, running rails console was triggering warnings.  However, those warnings look to be tied to a Gem version that has subsequently updated between start and finish of this PR.  I'll make a pass on those 3 additions and verify if they are still required or not.